noir/noir-repo/test_programs/execution_success/databus_two_calldata/src/main.nr

```diff
@@ -0,0 +1,5 @@
+fn main(mut idx: u32, y: call_data(0) [u32; 2], z: call_data(1) [u32; 4]) -> return_data u32 {
```

Comment:
This program results in a single databus calldata block constraint, with a calldata array of length 6. Expectation was two arrays, lengths 2 and 4.